### PR TITLE
feat: add spotlight animation

### PR DESCRIPTION
**Pull request by paulpopus**
CMS: https://github.com/payloadcms/website-cms/pull/66

Videos in slack

### CODE REVIEW

File: src/components/SpotlightAnimation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  }
+
+  return (
+    <div className={[classes.wrapper].filter(Boolean).join(' ')}>
+      <Element
+        style={{ backgroundPosition: getBackgroundOrigin }}
+        className={[classes.container, hasFadeIn && classes.hasFadeIn, ready && classes.ready]
+          .filter(Boolean)
+          .join(' ')}
+        // @ts-expect-error sorry
```
Comment (PatrikKozak) [inline src/components/SpotlightAnimation/index.tsx:111]: @paulpopus what's the reason for `@ts-expect-error` here?

File: src/components/SpotlightAnimation/types.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export type AS = Extract<keyof JSX.IntrinsicElements, 'p' | 'span' | 'h1' | 'h2' | 'h3'>
```
Comment (PatrikKozak) [inline src/components/SpotlightAnimation/types.ts:1]: @paulpopus Why did you name this type `AS` ? Should be something that makes sense so I can read the type and infer what the type probably is

File: src/components/SplitAnimate/index.module.scss [hunk 3]
```diff
@@ -1,8 +1,7 @@
 .word {
-  display: inline-block;
-  margin-right: 0.2em;
+  display: inline-flex;
   overflow: hidden;
-
+  margin-right: 0.16em;
```
Comment (PatrikKozak) [inline src/components/SplitAnimate/index.module.scss:4]: @paulpopus What's the need to specify such a specific margin here?

File: src/components/SpotlightAnimation/types.ts (hunk 2 above)
Comment (paulpopus) [reply src/components/SpotlightAnimation/types.ts]: renamed it to AllowedElements

**Pull request merged**